processor/k8sattributesprocessor/README.md

```diff
@@ -327,7 +327,7 @@ rules:
   resources: ["pods", "namespaces", "nodes"]
   verbs: ["get", "watch", "list"]
 - apiGroups: ["apps"]
-  resources: ["replicasets"]
+  resources: ["replicasets", "deployments"]
```

Comment:
Can we update the paragraph above that explains why/when these permissions are needed.

In addition, it would be nice to mention that the extra metadata come with an additional cost which comes as extra load to K8s API as well as the memory the Collector needs.